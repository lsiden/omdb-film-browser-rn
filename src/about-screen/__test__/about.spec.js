import React from "react"
import renderer from "react-native-test-utils"

import AboutScreen from "../"

it("renders list of titles", () => {
  const wrapper = renderer(<AboutScreen />)
  expect(wrapper.toJSON()).toMatchSnapshot()
})
