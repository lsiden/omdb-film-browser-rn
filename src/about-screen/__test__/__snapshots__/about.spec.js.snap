// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders list of titles 1`] = `
<RCTScrollView
  style={
    Object {
      "alignContent": "flex-start",
      "backgroundColor": "#05347f",
      "borderStyle": "solid",
      "borderTopColor": "white",
      "borderTopWidth": 2,
      "height": "100%",
      "paddingTop": 30,
    }
  }
>
  <View>
    <View
      style={
        Object {
          "alignItems": "center",
          "marginBottom": 15,
          "marginTop": 15,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "white",
            "fontSize": 14,
            "marginBottom": 10,
          }
        }
      >
        Written By:
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "white",
            "fontSize": 16,
            "marginBottom": 10,
          }
        }
      >
        Lawrence Siden
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        onPress={[Function]}
        style={
          Object {
            "color": "white",
            "fontSize": 16,
            "marginBottom": 10,
            "textDecorationLine": "underline",
          }
        }
      >
        Westside Consulting LLC
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        onPress={[Function]}
        style={
          Object {
            "color": "white",
            "fontSize": 16,
            "marginBottom": 10,
            "textDecorationLine": "underline",
          }
        }
      >
        westsideconsultingllc@gmail.com
      </Text>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "marginBottom": 15,
          "marginTop": 15,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "white",
            "fontSize": 14,
            "marginBottom": 10,
          }
        }
      >
        Credits:
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        onPress={[Function]}
        style={
          Object {
            "color": "white",
            "fontSize": 16,
            "marginBottom": 10,
            "textDecorationLine": "underline",
          }
        }
      >
        Create React Native App
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        onPress={[Function]}
        style={
          Object {
            "color": "white",
            "fontSize": 16,
            "marginBottom": 10,
            "textDecorationLine": "underline",
          }
        }
      >
        Open Movie Database
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        onPress={[Function]}
        style={
          Object {
            "color": "white",
            "fontSize": 16,
            "marginBottom": 10,
            "textDecorationLine": "underline",
          }
        }
      >
        Icons8
      </Text>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "marginBottom": 15,
          "marginTop": 15,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "white",
            "fontSize": 14,
            "marginBottom": 10,
          }
        }
      >
        Source code:
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        onPress={[Function]}
        style={
          Object {
            "color": "white",
            "fontSize": 16,
            "marginBottom": 10,
            "textDecorationLine": "underline",
          }
        }
      >
        react-native-omdb-viewer
      </Text>
    </View>
  </View>
</RCTScrollView>
`;
